Scripts/RemediationScripts/Remediate-SetDBForMySQLFlexibleServerMinReqTLSVersion.ps1

```diff
@@ -0,0 +1,819 @@
+
+                # Write this to a file.
+                $DBForMySQLFSRemediatedFile = "$($backupFolderPath)\RemediatedDBForMySQLFS.csv"
+                $DBForMySQLFSRemediated | Export-CSV -Path $DBForMySQLFSRemediatedFile -NoTypeInformation
+
+                Write-Host "This information has been saved to" -NoNewline
+                Write-Host " [$($DBForMySQLFSRemediatedFile)]" -ForegroundColor $([Constants]::MessageType.Update) 
+                Write-Host "Use this file for any roll back that may be required." -ForegroundColor $([Constants]::MessageType.Info)
+            }
+        
+            if ($($DBForMySQLFSSkipped | Measure-Object).Count -gt 0) {
+
+                Write-Host "Error while setting up the server parameter tls_version for Azure Database for MySQL flexible server(s) in the subscription:" -ForegroundColor $([Constants]::MessageType.Error)
+                Write-Host $([Constants]::SingleDashLine)
+                $DBForMySQLFSSkipped | Format-Table -Property $colsProperty -Wrap
+            
+                # Write this to a file.
+                $DBForMySQLFSSkippedFile = "$($backupFolderPath)\SkippedDBForMySQLFS.csv"
+                $DBForMySQLFSSkipped | Export-CSV -Path $DBForMySQLFSSkippedFile -NoTypeInformation
+                Write-Host "This information has been saved to"  -NoNewline
+                Write-Host " [$($DBForMySQLFSSkippedFile)]" -ForegroundColor $([Constants]::MessageType.Update)
+            }
+        }
+        if ($AutoRemediation) {
+            $logFile = "LogFiles\" + $($TimeStamp) + "\log_" + $($SubscriptionId) + ".json"
+            $log = Get-content -Raw -path $logFile | ConvertFrom-Json
+            foreach ($logControl in $log.ControlList) {
+                if ($logControl.ControlId -eq $controlIds) {
+                    $logControl.RemediatedResources = $logRemediatedResources
+                    $logControl.SkippedResources = $logSkippedResources
+                    $logControl.RollbackFile = $DBForMySQLFSRemediatedFile
+                }
+            }
+            $log | ConvertTo-json -depth 10  | Out-File $logFile
+        }
+    }
+    else {
+        Write-Host "[Step 4 of 4] Enable secure TLS version on Azure Database for MySQL flexible server(s) in the Subscription." 
+        Write-Host $([Constants]::SingleDashLine)
+        Write-Host "Skipped as -DryRun switch is provided." -ForegroundColor $([Constants]::MessageType.Warning)
+        Write-Host $([Constants]::SingleDashLine)
+        Write-Host "Next steps:" -ForegroundColor $([Constants]::MessageType.Warning)
+        Write-Host "Run the same command with -FilePath [$($backupFile)] and without -DryRun, to set secure TLS version for Azure Database for MySQL flexible server(s) listed in the file." -ForegroundColor $([Constants]::MessageType.Warning)
+        Write-Host $([Constants]::DoubleDashLine)
+    }
+}
+
+function Set-TLSVersiontoPreviousValueforDBForMySQLFlexibleServer {
```

Comment:
Isn't the convention we're following "Reset-..."
Let's try to follow the same format across because I saw in some places the command name as Set-X and Reset-X